### ISSUE TITLE
setLayerVisibility specified in documentation but not available in library

### ISSUE THREAD

**Issue by noorbakerally** [m0nac0/flutter-mapbox-gl#151]
The `setLayerVisibility` is documented in the API Documentation [here](https://htmlpreview.github.io/?https://raw.githubusercontent.com/m0nac0/flutter-maplibre-gl/docs/doc/api/maplibre_gl/MaplibreMapController/setLayerVisibility.html).

However, it's not available in the codes. I need to configure the visibility of certain map layers, such as `contour`, `summit`, `carparks`, etc. Is there another way to do this via the map controller?

**Comment by m0nac0** [comment]
This feature was merged in #138, and is not yet in a release. You need to use the main branch from this repo, are you doing that?

**Comment by noorbakerally** [comment]
Yes, I found this, I wasn't using the `main` branch, just to confirm, can we use this for hiding or displaying layers like `contour-line`, `mountain-peak`, `hike-route`, `bike-route-line`, etc?

**Comment by m0nac0** [comment]
Yes, if those layers exist you should be able to hide/show them.

**Comment by noorbakerally** [comment]
How about getting a layer object and modifying it's properties?

For example, 

I am adding a layer with id `tracker` as shown below. How do I get the layer object if I need to modify it layer, for example, changing the `iconSize` or `iconRotate` property. The `future` that addSymbolLayer returns does not contain the layer. Neither do I see any method like `getLayer` or `updateLayer`. Is there anything such?

```
 maplibreMapController.addSymbolLayer(
      "trackerSource",
      "tracker",
      SymbolLayerProperties(
          iconImage: "assets/images/NavIcon.png", //  "{type}-15",
          iconSize: 1,
          iconAllowOverlap: true,
          iconRotationAlignment: 'viewport',
          iconPitchAlignment: 'map',
      ),
    );
```

**Comment by m0nac0** [comment]
No this does not exist, yet. The challenge here is that maplibre on iOS does not support retrieving the JSON representation of a source or layer. I have some ideas how to implement this (updating is probably easier than retrieving), but I can't promise you when this will be ready. 

If someone is interested in working on this, please let me know and I can give you some more details.

**Comment by gabbopalma** [comment]
We apologize for responding to this issue after such a long time. It has been resolved over time.
